### ISSUE TITLE
.complicationset in xcassets isn't packaged correctly

### ISSUE THREAD

**Issue by BalestraPatrick** [bazelbuild/rules_apple#1364]
I'm comparing our Bazel and Xcode built apps and I've noticed some differences. We have a `watchos_extension` exactly like the multi-platform example in this repo: https://github.com/bazelbuild/rules_apple/blob/4246cfe864953025cdaa105d8105679fcd1fba29/examples/multi_platform/Buttons/BUILD#L184-L192

We're even specifying a `resources = glob(["Resources/Asset.xcassets/**"])` which contains a `.complicationset`  and a few `.imageset`. The `.imageset` are correctly compiled into a `Assets.car` inside the WatchExtension directory, but `.complicationset` aren't. Xcode does correctly include both into the compiled `Assets.car`. 

You can easily reproduce this by running the example in this repo (or using my branch that actually specifies `resources` directly): https://github.com/BalestraPatrick/rules_apple/tree/missing-complicationset-resources

```
$ bazelisk build //examples/multi_platform/Buttons
$ ls bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-dbb916128be5/bin/examples/multi_platform/Buttons/Buttons.app/Watch/ButtonsWatch.app/PlugIns/ButtonsWatchExtension.appex/Assets.car 
ls: bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-dbb916128be5/bin/examples/multi_platform/Buttons/Buttons.app/Watch/ButtonsWatch.app/PlugIns/ButtonsWatchExtension.appex/Assets.car: No such file or directory
```

I've tried to look through the codebase but couldn't find anything referencing a special handling of `.complicationset` so I would assume they should be treated in the same way as `.imageset`. Does anyone have more context on this?

**Comment by keith** [comment]
I just spun up an empty xcodeproj with this setup and it looks like there's a custom argument passed to actool for complications: `--complication Foo` where `Foo` is the name you see in the catalog. I guess we need to discover these directories and pass this as well.

**Comment by BalestraPatrick** [comment]
I verified in Xcode that the name of the complication has to be exactly "Complication". If it's anything else, actool will throw a warning:
>warning: The asset catalog does not contain a complication named "Complication".

Having multiple Complications seems to be unsupported as well.

So I think we can make the same assumption and if there's a `.complicationset`, we can pass `--complication Complication` to actool. I verified that this patch fixes it for me: https://github.com/bazelbuild/rules_apple/compare/master...BalestraPatrick:actool-watch-complication?expand=1

If you agree with this patch, I'd like some pointers to add a test. I'm not sure on the best place to add one. I couldn't find tests specific to `compile_asset_catalog`, but only some integration tests for the watch here: https://github.com/bazelbuild/rules_apple/blob/master/test/watchos_application_test.sh
